compute_modules/logging/common.py

```diff
@@ -66,10 +73,12 @@ class ComputeModulesLoggerAdapter(_LoggerAdapter):
     def __init__(
         self,
         logger_name: str,
+        handlers: Optional[Union[logging.Handler, Tuple[logging.Handler, ...]]],
+        formatter: Optional[logging.Formatter],
         process_id: int = -1,
         job_id: str = "",
     ) -> None:
-        self._p_logger = _create_logger(logger_name)
+        self._p_logger = _create_logger(logger_name, handlers, formatter)
```

Comment:
I "think" this should work with the way you have set up your adapter but will run some tests to check. Interested if you see any major immediate issues